### ISSUE TITLE
单独设置区域里的borderColor不起作用

### ISSUE THREAD

**Issue by liuxiaoqiang1018** [ecomfe/echarts-gl#497]
<template>
  <div class="home-map-root">
    <div class="home-map" ref="homeMap" id="homeMap">homeMap</div>
  </div>
</template>

<script>
import { qing_tao_map_json } from '../../json/map_json.js'
import 'echarts-gl';

export default {
  data() {
    return {
    }
  },
  created() {
  },
  mounted() {
    this.$echarts.registerMap("QingTaoMap", qing_tao_map_json);
    let mEcharts = this.$echarts.init(this.$refs.homeMap);
    let option = {
      geo3D: {
        // 将geo3d放在最底层
        zlevel: -1,
        map: 'QingTaoMap',
        regionHeight: 1,//区域的高度，通俗点说就是厚度

        itemStyle: {
          height: 1,
          color: '#2B5890', //地图颜色
          borderWidth: 0.5, //分界线wdith
          borderColor: '#333', //分界线颜色
        },
        label: {
          show: true, //是否显示label
          color: '#fff', //文字颜色
          fontSize: 20, //文字大小
        },
        regions: [
          {
            name: '即墨区', // 区域名称
            height: 2,
            itemStyle: {
              borderWidth: 10, // 设置该区域的边缘宽度
              //****这个设置不起作用 */
              borderColor: "#ff0",//设置该区域的边缘颜色
              color: '#f00',// 设置该区域的背景颜色
            },
          },
        ],
      },
      title: {
      },

      series: [
      ],
    };

    mEcharts.setOption(option,true);
  },
  methods: {
  },
}
</script>
<style scoped lang='less'>
.home-map-root {
  height: 100%;
  width: 100%;
}

.home-map {
  height: 100%;
  width: 17.5rem;
  // background-color: blueviolet;
}
</style>

**Comment by helgasoft** [comment]
yes, _textStyle_ for _geo3D.label_ and _geo3D.regions.label_ is redundant in v.5.5.0.
Workaround until fixed someday:  use attributes (like color,fontSize) directly, without  _textStyle_  - [Demo](https://echarts.apache.org/examples/en/editor.html?gl=1&code=LYTwwgFghgTgLgOgM4QPYHcAyqoBMCWAdgOYAUAlANwBQAJAsQKZykBKA8uwCoD6ACgEEuACQAEAalEByAPS4ocKDKhIkzGU1QzhAaQQArJKkJSANKIBmAV0IBjOPmOjSmgFJHC5UQG9qo0aCQsIgQ-LiM2HhEZFR-ooy20PBICDCMxPhIcIwwALJQAA6kUrpmom4esf6BSYhqcOwFDsakcf6kqE2OhKIAvD5t_qIOcAA2jABcPsOMAB5wU1KaAMwAIqKjUABGjKNIUqIAvqaD_iurU75D1wGFi6UnN0MwOMBTcDBWjI9PG9u7l1EKAw70-30sxjgAGV8AAvSaiAAc5lsqFGqBgiy2L3QJiOPyeaQyxiQUwA2qcbldftdCFBgAipABJJCbQi4fYEmmiCCMfDECALUQAVnMMhkohsuFQtisDMI2VwlKe4tERO6wj5AqFAEZRX1RNLGEhRIRUHBROgMQBrZU3fDZYBQuAgcaXO1PLYY8IwADqYTgECmACYxRKjSazRarTBbdybl6YD6wGiMVjRl8zB6bqj0ZjpAUiNapNn_MdS38dqN3fHruLsvNna6Ed4bqqI6bzZabRX_Lm09IthnGFna0NE8nU_mpGlcKOx6IJzl_bhA1MdVz4xZITD4evhb3ROLDhWT_Gz78ALqUi9l8hxWKHKhAA)

**Comment by zz925625** [comment]
ok;3q
